pkgs/misc/emulators/higan/0001-change-compiler.diff

```diff
@@ -0,0 +1,12 @@
+diff -rupN higan_v095-source.orig/nall/GNUmakefile higan_v095-source/nall/GNUmakefile
+--- higan_v095-source.orig/nall/GNUmakefile	2015-11-04 10:28:26.189750604 +0100
++++ higan_v095-source/nall/GNUmakefile	2015-11-04 10:28:31.752231593 +0100
+@@ -51,7 +51,7 @@ ifeq ($(compiler),)
+   else ifeq ($(platform),macosx)
+     compiler := clang++
+   else ifeq ($(platform),linux)
+-    compiler := g++-4.9
++    compiler := c++
```

Comment:
I think it would make sense to replace the compiler with c++ for all platforms, when compiling under Nix